dds/idl/marshal_generator.cpp

```diff
@@ -1315,6 +1303,13 @@ bool marshal_generator::gen_struct(AST_Structure*, UTL_ScopedName* name,
       if (first) be_global->impl_ << intro << "  return true;\n";
       else be_global->impl_ << intro << "  return " << expr << ";\n";
     }
+
+    be_global->header_ <<
+      "template <>\n"
+      "struct MarshalTraits<" << cxx << "> {\n"
+      "inline static bool gen_is_bounded_size() { return " << (is_bounded_struct ? "true" : "false") << "; }\n"
```

Comment:
"inline" isn't needed here, all member functions defined within the class definition are inline.  Please update in DDSTraits as well.